### ISSUE TITLE
/images/ view needs navigation (forward, back, save and advance) UI

### ISSUE THREAD

**Issue by simra** [Tech4Tracing/CartridgeOCR#68]
The old interface enabled image navigation and save+advance UI - this will be important for efficiency.

**Comment by simra** [comment]
Oh I see now the bulk annotate view might make this obsolete.  I'll play with it and see how it flows.

**Comment by koriaf** [comment]
Yes, my original idea is "bulk annotation", probably without any pagination - user navigates to collection and just annotates everything and then
* just presses big "Save" button which saves everything when done (so user can delete annotations and reset image if messed things up)
* the data for each image is saved automatically 10 seconds after user leaves image for the next one (and it's fine to press Save manually for the last one)

Although for large and complex images it might be better to annotate them one-by-one, for cases where user needs the whole screen area for it (if you put 30 items on your flatbed scanner for example)
